### ISSUE TITLE
Beta does not seem to affect entropy

### ISSUE THREAD

**Issue by AsadJeewa** [Unity-Technologies/ml-agents#2630]
I am using curiosity to find goals in large environments. I have had limited success. I run the experiments with the same set of parameters and the results are widely different every time. 

The major issue is that even when I increase Beta, My entropy keeps converging to 0 almost immediately.

I've adjusted various other hyper-parameters to attempt to solve my problems, but if the policy is deterministic, it becomes irrelevant.

The task is discrete and the agent must navigate to a far away goal. It can move in 8 directions.
The observations are agent and target position. +1 reward for finding goal, 0.0001 existential penalty.

![env](https://user-images.githubusercontent.com/27775903/65727257-7cdce880-e0b7-11e9-90e6-03eafcde0a27.PNG)
![beta isue](https://user-images.githubusercontent.com/27775903/65727258-7d757f00-e0b7-11e9-9489-1ce0bab007ca.PNG)
PARAM
    default:
        trainer: ppo
        batch_size: 1024
        beta: 5.0e-3
        buffer_size: 10240
        epsilon: 0.2
        gamma: 0.99
        hidden_units: 128
        lambd: 0.95
        learning_rate: 3.0e-4
        max_steps: 5.0e4
        memory_size: 256
        normalize: false
        num_epoch: 3
        num_layers: 2
        time_horizon: 64
        sequence_length: 64
        summary_freq: 1000
        use_recurrent: false
        use_curiosity: false
    
RollerBallBrainBranch:
        batch_size: 128
        buffer_size: 2048
        max_steps: 5.0e5
        use_curiosity: true
        beta: 1.0e-2
        reward_signals:
            extrinsic:
                strength: 1.0
                gamma: 0.99
            curiosity:
                strength: 0.1
                gamma: 0.99
                encoding_size: 64

Any other advice would be greatly appreciated

**Comment by AsadJeewa** [comment]
On certain instances, the agent is able to find the goal. On other runs, the agent either moves very small distances or keeps repeating the same poor policy. Each run is totally different.

**Comment by AsadJeewa** [comment]
Fixed through tedious and careful tuning of hyperparameters. Very sensitive: small changes result in huge difference in performance
